### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_EditAttachments_ScreenReader: Narrator does not announce anything when the attachments is 'deleted' or 'Restored'. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4016]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106 
Operating System:Windows10
**Steps to reproduce**
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Click on Run or right click and select Run to run the notebook.

4.Navigate to any file present under Files tab-->Hit enter-->View Menu-->Cell Toolbar-->Attachments. Click on Edit Attachments button appeared on page.
5. Start narrator.
6. Verify How narrator reads when the attachment is 'deleted' or 'restored'.

**Actual**
Narrator does not announce anything when the attachment is 'deleted' or 'restored'.

**Expected**
Narrator should announce the completion action whenever the attachment is deleted or restored.

**User Impact**
Screenreader user will not be able to know whether the action has been completed or not.
**Attachment:**
[A11y_AzureNotebooks_Jupyter_EditMetada_ScreenReader22.pptx](https://github.com/jupyter/notebook/files/2401324/A11y_AzureNotebooks_Jupyter_EditMetada_ScreenReader22.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
